docs/guides/sharded_ruler.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Config for horizontally scaling the Ruler"
+linkTitle: "Config for horizontally scaling the Ruler"
+weight: 4
+slug: ruler-sharding
+---
+
+## Context
+
+One option to scale the ruler is by scaling it horizontally. However, with multiple ruler instances running they will need to coordinate to determine which instance will evaluate which rule. Similar to the ingesters, the rulers establish a hash ring to divide up the responsibilities of evaluating rules. 
+
+## Config
```

Comment:
After rebasing and `make doc` you will also get the config file documentation updated. Could be worth adding a link to that doc (each root config block has an anchor, so you could link it to `#ruler-config`) mentioning it like `To see the complete set of config option, please check out...` 